src/schema/rules/entities.yaml

```diff
@@ -27,5 +27,6 @@
 - atlas
 - resolution
 - density
+- tracksys
```

Comment:
I would not put this so low. I would probably put it before or after `run`. Which seems a more sensible sorting?

```
sub-01_task-X_run-1_tracksys-Y_motion.tsv
sub-01_task-X_run-1_tracksys-Z_motion.tsv
sub-01_task-X_run-2_tracksys-Y_motion.tsv
sub-01_task-X_run-2_tracksys-Z_motion.tsv
```

```
sub-01_task-X_tracksys-Y_run-1_motion.tsv
sub-01_task-X_tracksys-Y_run-2_motion.tsv
sub-01_task-X_tracksys-Z_run-1_motion.tsv
sub-01_task-X_tracksys-Z_run-2_motion.tsv
```

Reply:
agreed 
especially that it seems to currently be with the entities that are more usually derivative entities

Reply:
agreed as well (although I remember that at some point I mentioned you may put the entity anywhere such as directly at the end).

> Which seems a more sensible sorting?

I think `tracksys` before `run` makes more sense, because I'd be looking for the different tracking systems, rather than how many runs of data are available and the ordering in your second example makes that easier to see.

Reply:
I also prefer putting it before `run` because of the way the mental search is typically done